pyseir/rt/constants.py

```diff
@@ -0,0 +1,75 @@
+from datetime import datetime
+
+import numpy as np
+
+
+class InferRtConstants:
```

Comment:
It would be cool if we could make this a dataclass with defaults. It'll make it easier to inject different values to test rather than needing to mock out the class.  Also, this could be `InferRtConfig`